feat(analytics-panel): 13240 - new analytics panel design

Akiyamka opened the pull request.

propakov committed:
feat(analytics-panel): 13240 - new analytics panel design

propakov committed:
feat(analytics-panel): 13240  - Apply new UI design of analytical panel

Akiyamka committed:
feat(analytics_panel): add real api add translation support

dqunbp commented on src/features/analytics_panel/components/AnalyticsDataList/AnalyticsDataList.tsx:
```diff
@@ -1,50 +1,54 @@
-const Sub = ({ children }: PropsWithChildren) => (
-  <span style={{ fontSize: '.7em', verticalAlign: 'super' }}>{children}</span>
-);
+const formatFractionalNumbers = (num: number) =>
```
Does this forrmatter meets the criteria:

_rounding: 2 decimal places, if the decimal part contains only zeros - don't display the decimal part_

For example we have `num = 2.004`

Then we will get `2.00`

Akiyamka replied:
fixed

Akiyamka committed:
fix: formating number by spec:
- 2 digits in fractional part
- show as integer when fractional part equal zero

Akiyamka committed:
chore: update translations

Akiyamka committed:
chore: sync translations
